bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -104,8 +102,50 @@
     private Set<String> bookiesToBeAudited = Sets.newHashSet();
     private volatile int lostBookieRecoveryDelayBeforeChange;
 
-    public Auditor(final String bookieIdentifier, ServerConfiguration conf,
-                   ZooKeeper zkc, StatsLogger statsLogger) throws UnavailableException {
+    static BookKeeper createBookKeeperClient(ServerConfiguration conf)
+            throws InterruptedException, IOException {
+        ClientConfiguration clientConfiguration = new ClientConfiguration(conf);
+        clientConfiguration.setClientRole(ClientConfiguration.CLIENT_ROLE_SYSTEM);
+        LOG.info("AuthProvider used by the Auditor is {}",
```

Comment:
I think this logging doesn't belong to this method. it should be kept at `initialize` method.

Reply:
nice catch. I addressed your comment in the latest change